### DISCUSSION TITLE
Any plans to add dropped handler?

### DISCUSSION THREAD

**Discussion by ezioda004** [uNetworking/uWebSockets.js#1011] [Q&A] [answered]
Hi,

As per [v20.49.0](https://github.com/uNetworking/uWebSockets/releases/tag/v20.49.0), there's a new dropped handler introduced. I checked in the latest build, and it's not there in the `WebSocketBehavior` class. Any plans to add the same?

**Comment by uNetworkingAB** [comment]
Of course it will be added. Do you need it?

**Reply by ezioda004** [reply]
Yes please. We’re using uwebsockets org wide and we track delivery of broadcasted messages.
Recently we’re seeing some clients are not receiving messages even though they are connected. I’m suspecting this is due to backpressure setting. Having a dropped handler would help with tracking this.

**Reply by uNetworkingAB** [reply]
Wait... it is already added?

**Reply by uNetworkingAB** [reply, marked as answer]
See latest release

**Reply by RJ4100** [reply]
How does the code look like to handle a dropped message which does not get delivered to some clients? @ezioda004 @uNetworkingAB
